### ISSUE TITLE
[1.1.26] ModAssistant says I am on 4.2.1 for BSIPA, installing claims it updates but it never does

### ISSUE THREAD

**Issue by alanmun** [Assistant/ModAssistant#461]
It appears to update on the UI, but when I close and reopen modassistant it is still 4.2.1 and says latest is 4.2.2.

**Comment by Melopod** [comment]
Have you launched the game yet? New installs don't get applied until you launch the game. All MA does is throw it in the folder. If you haven't launched the game the only version that MA will be able to detect as properly installed will be the older version.

**Comment by alanmun** [comment]
> Have you launched the game yet? New installs don't get applied until you launch the game. All MA does is throw it in the folder. If you haven't launched the game the only version that MA will be able to detect as properly installed will be the older version.

Yes, but I am on the newest version of beat saber if that matters. I also cannot see the modding menu in game. Is it just because I’m on the newest version of BS?

**Comment by Melopod** [comment]
Sounds like a more generic modding issue, this has nothing to do with MA. I would recommend troubleshooting via the wiki (bsmg.wiki) or the help channels in the Beat Saber Modding Group Discord (discord.gg/beatsabermods)

**Comment by MagnusWestin** [comment]
See issue 451, that solve it for me. I uninstalled, validated the game, it replaced 10 files. And then installed BSIPA again. Now it works.

**Comment by scowalt** [comment]
I hit this issue. Uninstalling all mods, validating games files, and installing all my mods again didn't work.

I think it worked after a second time when I chose to install a different set of mods.

So the _real_ issue may be that one of the mods on the list isn't compatible with game version 1.22.1, and when that mod fails to install on launch, all of the other mods do as well?

**Comment by N4T8281** [comment]
I've started having this issue too, I've tried a range of installations and uninstalls, selecting different mods alongside BSIPA, the version issue only shows up on the specific BSIPA file though. No error message about failed installs either.
